examples/k3s-setup-new/terraform.tfvars.example

```diff
@@ -0,0 +1,2 @@
+metal_api_token="your_token_here" #This must be a user API token
```

Comment:
Just curious, do you know why this has to be a user token?  Long-term we may be able to add the missing capabilities to project tokens, depending on what's missing.

Reply:
This is the `auth_token` required by the [equinix provider](https://registry.terraform.io/providers/equinix/equinix/latest/docs#auth_token). My understanding is that the usr token is necessary there, unless this has changed?